docs/examples/auth_manage_projects/list_and_create_projects.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python
+
+import argparse
+import os
+
+import globus_sdk
+from globus_sdk.tokenstorage import SimpleJSONFileAdapter
+
+MY_FILE_ADAPTER = SimpleJSONFileAdapter(
+    os.path.expanduser("~/.sdk-manage-projects.json")
+)
+
+SCOPES = [globus_sdk.AuthClient.scopes.manage_projects, "openid", "email"]
+RESOURCE_SERVER = globus_sdk.AuthClient.resource_server
+
+# tutorial client ID
+# we recommend replacing this with your own client for any production use-cases
+CLIENT_ID = "61338d24-54d5-408f-a10d-66c06b59f6d2"
+
+NATIVE_CLIENT = globus_sdk.NativeAppAuthClient(CLIENT_ID)
+
+
+def do_login_flow():
+    NATIVE_CLIENT.oauth2_start_flow(requested_scopes=SCOPES)
+    authorize_url = NATIVE_CLIENT.oauth2_get_authorize_url()
+    print(f"Please go to this URL and login:\n\n{authorize_url}\n")
+    auth_code = input("Please enter the code here: ").strip()
+    tokens = NATIVE_CLIENT.oauth2_exchange_code_for_tokens(auth_code)
+    return tokens.by_resource_server[RESOURCE_SERVER]
+
+
+def get_auth_client():
+    tokens = do_login_flow()
+    return globus_sdk.AuthClient(
+        authorizer=globus_sdk.AccessTokenAuthorizer(tokens["access_token"])
+    )
+
+
+def create_project(args):
+    auth_client = get_auth_client()
+    userinfo = auth_client.oauth2_userinfo()
+    print(
+        auth_client.create_project(
+            args.name,
+            contact_email=userinfo["email"],
+            admin_ids=userinfo["sub"],
+        )
+    )
+
+
+def list_projects(args):
```

Comment:
Is `args` included as a parameter for easier extensibility? It doesn't make sense to me but could perhaps be a jumping-off point for new developers.

Reply:
Extensibility and uniformity (since the other methods also consume `args`), but I could happily drop it if you think it's more likely to cause confusion than dispel it.

My goal here is less to write a truly exemplary script than it is to make something which is "good enough" and which will be approachable to the majority of our (python-inclined) users. The Auth logic is sufficiently intense to teach/learn/read that I think _anything_ which distracts from that should be removed.

So is it better to pass `args` to each method, even though it's useless in this case, or to omit it in this case but pass it in the others? Since you commented, it sounds to me like you're inclined towards removing it. I'll make that change, but let me know if you think it should be shaped differently or if there are other opportunities to simplify the example.

Reply:
I do lean toward removing it. If I opened the file (or copied/pasted it) in a competent IDE and saw code lint warnings I would wonder about the quality and utility of the code. Thanks for addressing this!